k8s/mongodb/mongo-data-configdb-pvc.yaml

```diff
@@ -0,0 +1,17 @@
+##########################################################
+# This YAML file desribes a k8s pvc for mongodb configDB #
+##########################################################
+
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: mongo-configdb-claim
+  annotations:
+    volume.beta.kubernetes.io/storage-class: slow-configdb
+spec:
+  accessModes:
+  - ReadWriteOnce
+  persistentVolumeReclaimPolicy: Retain
```

Comment:
From https://kubernetes.io/docs/user-guide/persistent-volumes/.
> Currently, only NFS and HostPath support recycling. AWS EBS, GCE PD, Azure Disk, and Cinder volumes support deletion.

I can specify it, but the UI still says that the reclaim policy is `Deletion`